vendor/diff-parsers-ext-parsewyc.patch

```diff
@@ -0,0 +1,948 @@
+       | _ ->
+           None
+     )
++*)
+ 
+ }
+ 
+ let newline = ('\013'* '\010')
+ let blank = [' ' '\009' '\012']
+@@@@
+                comment lexbuf)
+             lexbuf
+         in
+         COMMENT (s, loc) }
+   | "(*)"
+-      { if !print_warnings then
+-          Location.prerr_warning (Location.curr lexbuf) Warnings.Comment_start;
++      { (*if !print_warnings then
++          Location.prerr_warning (Location.curr lexbuf) Warnings.Comment_start;*)
+         let s, loc = wrap_comment_lexer comment lexbuf in
+         COMMENT (s, loc) }
+   | "(*" (('*'*) as stars) "*)"
+       { if !handle_docstrings && stars="" then
+          (* (**) is an empty docstring *)
+           DOCSTRING(Docstrings.docstring "" (Location.curr lexbuf))
+         else
+           COMMENT (stars, Location.curr lexbuf) }
+   | "*)"
+-      { let loc = Location.curr lexbuf in
+-        Location.prerr_warning loc Warnings.Comment_not_end;
++      { (*let loc = Location.curr lexbuf in
++        Location.prerr_warning loc Warnings.Comment_not_end;*)
+         lexbuf.Lexing.lex_curr_pos <- lexbuf.Lexing.lex_curr_pos - 1;
+         let curpos = lexbuf.lex_curr_p in
+         lexbuf.lex_curr_p <- { curpos with pos_cnum = curpos.pos_cnum - 1 };
+         STAR
+       }
+@@@@
+   | '\\' _
+       { if not (in_comment ()) then begin
+ (*  Should be an error, but we are very lax.
+           error lexbuf (Illegal_escape (Lexing.lexeme lexbuf, None))
+ *)
+-          let loc = Location.curr lexbuf in
+-          Location.prerr_warning loc Warnings.Illegal_backslash;
++          (*let loc = Location.curr lexbuf in
++          Location.prerr_warning loc Warnings.Illegal_backslash;*)
+         end;
+         store_lexeme lexbuf;
+         string lexbuf
+       }
+   | newline
+-      { if not (in_comment ()) then
+-          Location.prerr_warning (Location.curr lexbuf) Warnings.Eol_in_string;
++      { (*if not (in_comment ()) then
++          Location.prerr_warning (Location.curr lexbuf) Warnings.Eol_in_string;*)
+         update_loc lexbuf None 1 false 0;
+         store_lexeme lexbuf;
+         string lexbuf
+       }
+   | eof
+--- ocaml-4.13-extended/parser.mly
++++ parse-wyc/lib/parser.mly
+@@@@
+    text comprised between the markers [BEGIN AVOID] and -----------
+    [END AVOID] has been removed. This file should be formatted in
+    such a way that this results in a clean removal of certain
+    symbols, productions, or declarations. */
+ 
++/* parse-wyc:
++   Compared to upstream, rules with errors on the RHS (between [BEGIN AVOID]
++   and [END AVOID]) are commented. Some rules definition are kept to minimize
++   the diff as long as the call sites are commented. This is necessary to
++   trigger the recovery in case of invalid input, instead of triggering the
++   error rules. */
++
+ %{
+ 
+ open Asttypes
+ open Longident
+ open Parsetree
+ open Ast_helper
+ open Docstrings
+ open Docstrings.WithMenhir
++open Let_binding
```

Comment:
Is this diff (aka adding let_binding.ml) necessary ?

Reply:
Yes it's something I tried to fix in the past, but I get an error in the generated `parser.mli` file: `Unbound type constructor let_bindings`, even with the same flags in the dune file.